packages/react/src/patterns/blocks/CalloutWithMedia/README.md

```diff
@@ -0,0 +1,72 @@
+# Callout With Media
+
+> The "Callout With Media" is a decorator of `Callout`, which includes
+> `Content Block Simple`.
+
+## Getting started
+
+Here's a quick example to get your started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+@import '@carbon/ibmdotcom-styles/scss/patterns/blocks/callout-with-media/index';
+```
+
+> 💡 Only import fonts once per usage. Don't forget to import the
+> CalloutWithMedia styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+```javascript
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { CalloutWithMedia } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+function App() {
+  const copy = 'lorum ipsum dolor sit ameet';
+  const heading = 'lorum ipsum dolor sit amet';
+  const image = 'image';
+  const mediaData = .... image data json...
```

Comment:
sorry, I meant to be fairly explicit for this one: 
```
const mediaType = 'image';
const mediaData = {
      heading: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
      image: {
        sources: [
          {
            src: 'https://dummyimage.com/320x180/ee5396/161616&text=16:9',
            breakpoint: 320,
          },
          {
            src: 'https://dummyimage.com/400x225/ee5396/161616&text=16:9',
            breakpoint: 400,
          },
          {
            src: 'https://dummyimage.com/672x378/ee5396/161616&text=16:9',
            breakpoint: 672,
          },
        ],
        alt: 'Image alt text',
        defaultSrc: 'https://dummyimage.com/672x378/ee5396/161616&text=16:9',
      },
    }
```

Reply:
ah my fault